memory/src/main/java/org/opensearch/ml/memory/action/conversation/CreateInteractionTransportAction.java

```diff
@@ -86,14 +88,20 @@ protected void doExecute(Task task, CreateInteractionRequest request, ActionList
         String rsp = request.getResponse();
         String ogn = request.getOrigin();
         String prompt = request.getPromptTemplate();
-        String additionalInfo = request.getAdditionalInfo();
+        Map<String, String> additionalInfo = request.getAdditionalInfo();
+        String parintid = request.getParent_interaction_id();
```

Comment:
`getParent_interaction_id` is not a java style method name. change to `getParentInteractionId` ?

Reply:
good catch. I think this is coming from the name convention from the index that is not corrected yet in the code.